src/Sql/Sql/Database/Cmdlet/NewAzureSqlDatabase.cs

```diff
@@ -159,6 +159,31 @@ public class NewAzureSqlDatabase : AzureSqlDatabaseCmdletBase<AzureSqlDatabaseCr
             Management.Sql.Models.DatabaseLicenseType.BasePrice)]
         public string LicenseType { get; set; }
 
+        /// <summary>
+        /// Gets or sets the compute model for Azure Sql database
+        /// </summary>
+        [Parameter(ParameterSetName = VcoreDatabaseParameterSet, Mandatory = false,
+            HelpMessage="The compute model for Azure Sql database")]
+        [PSArgumentCompleter(
+            "Provisioned",
+            "Serverless")]
+        public string ComputeModel { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Auto Pause delay for Azure Sql Database
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The auto pause delay for Azure Sql database(serverless only), -1 to opt out")]
+        public int? AutoPauseDelay { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Minimal capacity that database will always have allocated, if not paused
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The Minimal capacity that database will always have allocated, if not paused. For Azure Sql database serverless only.")]
+        [Alias("MinVCore")]
+        public double? MinCapacity { get; set; }
```

Comment:
I prefer making `Capacity` the primary name of this, so you can leave this as-is.

Reply:
One way is to use MinVCore as default name, and MinCapacity as alias, but API actually have MinCapacity parameter which will be confusing.
Or I can change vise versa: Capacity as default parameter and Vcore as alias, which is more logical since GetDatabase have capacity in result, and don't have Vcore. Is it breaking change?

Reply:
You are exactly right: API actually has Capacity, so we should stick with that terminology. Leave this as MinCapacity, and later (separate PR) we can change VCore to Capacity. As long as we use alias it's not a breaking change.